pybandits/quantitative_model/base.py

```
@@ -0,0 +1,249 @@
+        Parameters
+        ----------
+        code : str
+            python code representing a function or a callable object.
+
+        Returns
+        -------
+        Callable
+            The deserialized function or callable object.
+        """
+        if code.startswith("def "):
+            exec(code, globals())
```

Ruff (0.15.15)

[error] Use of `exec` detected

(S102)